packages/gatsby/src/utils/__tests__/get-ssl-cert.js

```diff
@@ -11,21 +11,21 @@ jest.mock(`gatsby-cli/lib/reporter`, () => {
     info: jest.fn(),
   }
 })
-jest.mock(`devcert-san`, () => {
+jest.mock(`devcert`, () => {
   return {
-    default: jest.fn(),
+    certificateFor: jest.fn(),
   }
 })
 
-const devcertSan = require(`devcert-san`).default
+const certificateFor = require(`devcert`)
```

Comment:
This should either be destructured, or aliased.  Right now it's importing as a namespace.  This will likely make the test fail further down.

`devcert` is written in typescript, and using `modules.export = function()` is generally frowned upon; it uses named exports for all of its functions.  So this namespace won't be a function you can call; instead it'll be an object containing the functions.  You'll notice that the `jest.mock('devcert')` module above is set up to mirror `devcert`'s structure.  It returns an object with a function, not a function.

You could do one of these:
```ts
// destructure the import
const { certificateFor } = require(`devcert`);
// or, in keeping with the source get-ssl-cert.js
const certificateFor = require(`devcert`).certificateFor;
// If you wanted the function name to fit its purpose a bit more clearly, you could alias it
const getDevCert = require(`devcert`).certificateFor;
```